app/assets/stylesheets/pages/_index.scss

```diff
@@ -1,2 +1,6 @@
 // Import page-specific CSS files here.
 @import "home";
+
+.block_radio_button_collection {
```

Comment:
Lucien what's this?
This page should only contain '@import "components_name";', no actual css code.
Maybe you want to add a file into components for the radio-buttons!